### PR TITLE
Friendapi

### PR DESCRIPTION
**Pull request by jsmith465264**

### CODE REVIEW

File: unhindled/templates/unhindled/friends.html [hunk 1]
```diff
@@ -6,25 +6,27 @@
...
 {% /header %}
+{% load friends %}
 
 <main>
 <p class="non-white-title">Manage Friends</p>
 
 <form action="/follow" method="POST">
 	{% csrf_token %}
-	<label for="author">Friends Username: </label>
+	<label for="author">Friends display name: </label>
```
Comment (PhamilyExpress) [inline unhindled/templates/unhindled/friends.html:16]:
```suggestion
	<label for="author">Friends Display Name: </label>
```
just pedantic

File: unhindled/templates/unhindled/friends.html [hunk 2]
```diff
@@ -6,25 +6,27 @@
...
 	<input id="author" type="text" name="author" value="">
 	<input type="hidden" name="next" value="{{ request.path }}">
     	<input type="submit" value="OK">
 </form>
 {% if object_list %}
 	{% for follow in object_list %}
-	        {% if follow.follower.username == user.username %}
+	        {% if follow.follower.toString == user.id.toString %}
```
Comment (PhamilyExpress) [inline unhindled/templates/unhindled/friends.html:23]: ?

Comment (PhamilyExpress) [reply unhindled/templates/unhindled/friends.html:23]: a bit unclear maybe

Comment (jsmith465264) [reply unhindled/templates/unhindled/friends.html:23]: follower is now a uuid instead of an object so comparing ids rather than usernames.

**Pull request merged**